app/javascript/gobierto_dashboards/index.js

```diff
@@ -1,16 +1,33 @@
 import "../../assets/stylesheets/module-dashboards.scss"
 
 import { ContractsController } from "./modules/contracts_controller.js";
+import { SubsidiesController } from "./modules/subsidies_controller.js";
 
 document.addEventListener('DOMContentLoaded', () => {
-  const appNode = document.getElementById("gobierto-dashboards-contracts-app");
+  const contractsAppNode = document.getElementById("gobierto-dashboards-contracts-app");
 
-  new ContractsController({
-    siteName: appNode.dataset.siteName,
-    logoUrl: appNode.dataset.logoUrl,
-    homeUrl: appNode.dataset.homeUrl,
-    contractsEndpoint: appNode.dataset.contractsEndpoint,
-    tendersEndpoint: appNode.dataset.tendersEndpoint,
-    dataDownloadEndpoint: appNode.dataset.dataDownloadEndpoint,
-  });
+  if (contractsAppNode) {
+    new ContractsController({
+      siteName: contractsAppNode.dataset.siteName,
+      logoUrl: contractsAppNode.dataset.logoUrl,
+      homeUrl: contractsAppNode.dataset.homeUrl,
+      contractsEndpoint: contractsAppNode.dataset.contractsEndpoint,
+      tendersEndpoint: contractsAppNode.dataset.tendersEndpoint,
+      dataDownloadEndpoint: contractsAppNode.dataset.dataDownloadEndpoint,
+    });
+  }
+});
+
+document.addEventListener('DOMContentLoaded', () => {
+  const subsidiesAppNode = document.getElementById("gobierto-dashboards-subsidies-app");
```

Comment:
There's no need to declare the same listener twice

Reply:
🤦